.eslintrc.json

```diff
@@ -6,15 +6,14 @@
     },
     "plugins": [
         "@typescript-eslint",
-        "react",
-        "react-hooks",
         "prettier",
         "patternfly-react"
     ],
     "extends": [
         "eslint:recommended",
         "plugin:@typescript-eslint/recommended",
-        "plugin:react/recommended"
+        "plugin:react/recommended",
+        "plugin:react-hooks/recommended"
```

Comment:
Although this plugin was already provided before, it was never extended. This caused regressions in the enforcement of the 'Rules of Hooks'.

Now that it is included it should no longer be possible to accidentally re-introduce a regression in this regard.